Mathlib/RingTheory/Localization/AtMinimalPrime.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2024 Uni Marx. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Uni Marx
+-/
+import Mathlib.RingTheory.Localization.AtPrime
+import Mathlib.RingTheory.Nilpotent
+import Mathlib.RingTheory.Ideal.MinimalPrime
+
+/-!
+# Localization at minimal prime ideals
+## Main results
+- `AtMinimalPrime.prime_unique`: When localizing at a minimal prime ideal `I`, the resulting
+  ring only has a single prime ideal.
+## Tags
+localization, prime, minimal, nilpotent
+-/
+
+variable {R : Type*} [CommSemiring R] (M : Submonoid R) (S : Type*) [CommSemiring S]
```

Comment:
Why keeping these variables outside the section and the following ones inside it?